frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -358,6 +358,22 @@ export default function SurveyEdit(): JSX.Element {
                                                                                   textPlaceholder="ex: We really appreciate it."
                                                                               />
                                                                           </LemonField.Pure>
+                                                                          <LemonField.Pure label="Button text">
+                                                                              <LemonInput
+                                                                                  value={
+                                                                                      survey.appearance
+                                                                                          .thankYouMessageCloseButtonText
+                                                                                  }
+                                                                                  onChange={(val) =>
+                                                                                      setSurveyValue('appearance', {
+                                                                                          ...survey.appearance,
+                                                                                          thankYouMessageCloseButtonText:
+                                                                                              val,
+                                                                                      })
+                                                                                  }
+                                                                                  placeholder="ex: Close"
```

Comment:
I'd prefer spelling this out explicitly. This aligns better with our tone of voice and improves clarity.

```suggestion
                                                                                  placeholder="example: Close"
```